base/version.jl

```diff
@@ -139,7 +139,7 @@ function tryparse(::Type{VersionNumber}, v::AbstractString)
     end
     prerl = prerl !== nothing ? split_idents(prerl) : minus !== nothing ? ("",) : ()
     build = build !== nothing ? split_idents(build) : plus  !== nothing ? ("",) : ()
-    return VersionNumber(major, minor, patch, prerl::VerTuple, build::VerTuple)
+    return @inline VersionNumber(major, minor, patch, prerl::VerTuple, build::VerTuple)
```

Comment:
I don't think we need this change any more - it was fixed a long time ago by de-specializing the constructor